docs/README.md

```diff
@@ -0,0 +1,224 @@
+What is this software?
+----------------------
+
+This is the MPI Testing Tool (MTT) software package.  It is a
```

Comment:
I believe we moved away from calling this the "MPI" testing tool when we started building the Python client as it became more of a generic capability. I've heard suggestions for "Middleware" testing tool, though I think we should throw it open to broader consideration. Certainly "MPI" is no longer really appropriate.